projects/frontend/shared-components/gui/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "shared-wrapper",
+  "version": "0.0.0",
```

Comment:
what's the version of the wrapper? I think it was 3.0.0

Reply:
Since we are having a unique artifact prefix `@vdk`, maybe we need to reconsider and start from 1 or something.
Not sure yet if we need version.txt file explicitly

Reply:
I don't think it matter that much, to be honest. It depends on if we consider it a new package or open-sourcing and existing package. But I'm ok with doing either. I've currently set both package.json files to point to version 3.0.0

Reply:
I'll set the version to `0.0.1` following your example here https://github.com/vmware/versatile-data-kit/pull/1626/files#diff-b2c4b01f8643ec76b11a2a2dcfc3903d534023cda3ee4b90dc1f8719b9b43dd4